gltfTutorial/gltfTutorial_010_Materials.md

```diff
@@ -0,0 +1,40 @@
+Previous: [Meshes](gltfTutorial_009_Meshes.md) | [Table of Contents](README.md) | Next: [Simple Material](gltfTutorial_011_SimpleMaterial.md)
+
+# Materials
+
+## Introduction
+
+The purpose of glTF is to define a transmission format for 3D assets. As shown in the previous sections, this includes information about the scene structure and the geometric objects that appear in the scene. But a glTF asset can also contain information about the *appearance* of the objects; that is, how these objects should be rendered on the screen.
+
+There are several well-known parameters that are commonly used to describe a material in rendering applications and graphics APIs:
+
+* The **diffuse** color: The color of the light that is diffusely reflected by the material. In the simplest form, this can be imagined as the "main" color that the material appears to have. It is often defined by a texture.
+* The **specular** color: The color of reflected highlights.

```

Comment:
I noticed this, particularly because the next one also refers to "specular highlights". 

One could argue here: These terms, `specular` etc., are basically just "placeholders for concepts", and quite often they are simply variable names. 

For the specific point, one could probably just say "The color of highlights".